remill/OS/OS.h

```diff
@@ -24,7 +24,7 @@
 #   define REMILL_ON_LINUX 0
 #   define REMILL_ON_WINDOWS 0
 #   define REMILL_OS "macos"
-# elif defined(__linux__)
+# elif defined(__linux__) || defined(EMSCRIPTEN)
```

Comment:
I was looking for this definition, but I couldn't find it in this PR. Is this a non documented value provided by the toolchain? Docs: https://emscripten.org/docs/compiling/Building-Projects.html#detecting-emscripten-in-preprocessor

Reply:
Yeah it looks like it's undocumented, but a good find since they now have the documented version __EMSCRIPTEN__. I'll change it to using that :)

As a word of caution, Emscripten's documentation is notoriously bad. I think it comes from how fast that project evolved that the written documentation was quickly broken by new features. For the source of truth I usually clone the emscripten repo and search through it rather than look at docs:

https://github.com/emscripten-core/emscripten/blob/430c2911dbe0104beab264cd19a971a2a7c6b793/emcc.py#L1264